assets/models/system/gpt2-large/spec.yaml

```diff
@@ -4,8 +4,8 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 97935fc1a406f447320c3db70fe9e9875dca2595
-  evaluation-min-sku-spec: 16|0|56|112
-  evaluation-recommended-sku: Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3, Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3, Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  evaluation-min-sku-spec: 4|0|28|56
```

Comment:
cpu-mem and disk space has reduced for this. Do we need to validate on in-min compute for this one?

Reply:
Validation exp - https://ml.azure.com/experiments/id/826c38b2-9d5a-4261-ba56-57b918a6729f?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-model-evaluation-rg/providers/Microsoft.MachineLearningServices/workspaces/aml-shared-model-evaluation-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47